BUILDING.md

```diff
@@ -101,6 +101,13 @@ File: `C:/Users/<username>/AppData/Roaming/Code/User/settings.json`
 }
 ```
 
+### Rust
```

Comment:
I would move this above the VS code steps, as a subsection of `MSYS2`, since this is specific to msys2, other environments, like a pure win32 setup with a native Rust, will have different steps...